automatic-gensyms.lisp

```diff
@@ -0,0 +1,187 @@
+;;;; -*- Mode: LISP; Syntax: COMMON-LISP; Package: LET-OVER-LAMBDA; Base: 10 -*- file: let-over-lambda.lisp
+
+;;;; Copyleft Spenser Truex "Equwal" 2018
```

Comment:
Could you remove the "copyleft" declaration, please? It's an unfair limitation of license that violates the intent of the original release, of which this is a derivative work.

Reply:
My mistake.